spec/models/medical_safety_alert_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+describe MedicalSafetyAlert do
+
+  def medical_safety_alert_content_item(n)
+    {
+      "content_id" => SecureRandom.uuid,
+      "base_path" => "/drug-device-alerts/example-medical-safety-alert-#{n}",
+      "title" => "Example Medical Safety Alert #{n}",
+      "description" => "This is the summary of example Medical Safety Alert #{n}",
+      "format" => "specialist_document",
+      "publishing_app" => "specialist-publisher",
+      "rendering_app" => "specialist-frontend",
+      "locale" => "en",
+      "phase" => "live",
+      "public_updated_at" => "2015-11-16T11:53:30",
+      "publication_state" => "draft",
+      "details" => {
+        "body" => "## Header" + ("\r\n\r\nThis is the long body of an example Medical Safety Alert" * 10),
```

Comment:
Also of note, this does *need* to be Govspeak right now, as this is what the User is editing every time they edit the document.